tokio-util/Cargo.toml

```diff
@@ -43,8 +43,6 @@ futures-util = { version = "0.3.0", optional = true }
 pin-project-lite = "0.2.11"
 slab = { version = "0.4.4", optional = true } # Backs `DelayQueue`
 tracing = { version = "0.1.29", default-features = false, features = ["std"], optional = true }
-
-[target.'cfg(tokio_unstable)'.dependencies]
 hashbrown = { version = "0.14.0", default-features = false, optional = true }
```

Comment:
Hmm. I'll have to think about what implications this has for our MSRV.

Reply:
hashbrown 0.14.0 has a `rust-version = "1.64.0"`.
hashbrown 0.15.2 has a `rust-version = "1.65.0"`.

It doesn't seem to have an explicit MSRV policy, but it is inline with tokio-util's 1.70.0

Reply:
I'm okay with this, but I don't want to require hashbrown if you're just using the `TaskTracker`. Can you add a `join_map` feature?

Reply:
That seems reasonable. Will do